extensions/amp-story/0.1/animation-presets.js

```diff
@@ -25,6 +26,44 @@ import {
 } from './animation-presets-utils';
 import {px} from '../../../src/style';
 
+/**
+ * A list of panning animation names.
+ * @private @const {!Array<string>}
+ */
+const PANNING_ANIMATION_NAMES = [
+  'pan-up',
+  'pan-down',
+  'pan-right',
+  'pan-left',
+];
+
+/**
+ * A mapping of animation names to CSS class names.
+ * @private @const {!Object<string, string>}
+ */
+const ANIMATION_CSS_CLASS_NAMES = {
+  'pan': 'i-amphtml-story-grid-template-with-pan-animation',
+};
+
+/**
+ * Perform style-specific operations for presets.
+ * @param {!Element} el
+ * @param {string} presetName
+ */
+export function setStyleForPreset(el, presetName) {
+  const pan = 'pan';
+  const fillTemplate = 'fill';
+
+  // For panning and zooming animations.
+  if (PANNING_ANIMATION_NAMES.includes(presetName)) {
```

Comment:
I'm not sure we can use `includes`; can you use `PANNING_ANIMATION_NAMES.indexOf(presetName) >= 0`?

Reply:
Sure. Out of curiosity, why wouldn't we want to use `includes`?

Reply:
It might be okay?  I'm not 100% sure.  I thought I remembered that some things that are added with es6 are not transpiled, but I never know which ones.